main.go

```diff
@@ -223,6 +223,15 @@ func init() {
 		config.SMTP = true
 	}
 
+	if config.SMTP && !config.EHLO {
+		name, err := os.Hostname()
+		if err != nil {
+			zlog.Errorf("Unable to set hostname")
```

Comment:
nit: This should be `Fatalf`, also  write error messages with leading lowercase. Also, this is unable to _get_ hostname, not set.
```
zlog.Fatalf("unable to get hostname for EHLO: %s", err.Error())
```